README.md

```diff
@@ -35,6 +58,36 @@ This shell script contains commands that build and runs the fala app.
 
 If you're not using `Docker Desktop`, you may want to have two terminals open.
 
+### PlayWright Testing with Docker
+
+Playwright has its own Docker container. Tests are not executed in the FALA Docker container but instead inside a 
+Playwright Docker container, which looks for FALA's URLs to test against.
+
+You will need the FALA Docker container running in order for the tests to work, as the Playwright Docker container 
+interacts with FALA's URLs.
+
+To spin up the Playwright Docker container, do the following:
+`docker-compose up -d --build playwright`
+
+The Playwright Docker container executes all Playwright tests within the playwright directory of the project. 
+Once the tests are complete, the container will shut itself down.
+
+If you want to keep the Playwright Docker container running and use it alongside your development environment, 
+you can do the following:
+`docker-compose run --entrypoint sh playwright`
+
+Tip: Remember to type `bash` first when interacting with a Docker container.
+
+This allows you to bash into the Playwright Docker container. From there, you can change the directory to `fala_local`:
+`cd ../fala_local/playwright/`
+
+`fala_local` directory reflects your local IDE changes in the Playwright container. Here, you can interact with the 
+command line to run your tests as many times as you like without having to delete and rebuild the Playwright Docker 
+container.
+
+To run tests inside the playwright directory:
+`pytest`
```

Comment:
I think it's worth making clearer that you should run `pytest`, when you've bashed into the Docker container. Wasn't clear upon first reading.

Reply:
Cool, I'll add that. I've added a new header for this section to help split up the two actions you can do with the playwright docker container.